File: src/main/java/fi/metatavu/metaform/server/metaforms/MalformedVersionJsonException.java
package fi.metatavu.metaform.server.metaforms;
/**
 * Exception throw when malformed version data json
 * 
 * @author Tianxing Wu
 */
public class MalformedVersionJsonException extends Exception {


  private static final long serialVersionUID = 6760138346241227829L;

  /**
   * Constructor
   *
   * @param message message
   * @param original original exception
   */
  public MalformedVersionJsonException(String message, Throwable original) {
    super(message, original);
  }

  /**
   * Constructor
   *
   * @param message message
   */
  public MalformedVersionJsonException(String message) {
    super(message);
  }
  
}

File: src/main/kotlin/fi/metatavu/metaform/server/exceptions/MalformedVersionJsonException.kt
package fi.metatavu.metaform.server.exceptions

/**
 * Exception throw when malformed version data json
 *
 * @author Tianxing Wu
 */
class MalformedVersionJsonException : Exception {
    /**
     * Constructor
     *
     * @param message message
     * @param original original exception
     */
    constructor(message: String?, original: Throwable?) : super(message, original)

    /**
     * Constructor
     *
     * @param message message
     */
    constructor(message: String?) : super(message)

    companion object {
        private const val serialVersionUID = 6760138346241227829L
    }
}